v3/primitives/pushbutton.py

```diff
@@ -26,7 +26,7 @@ def __init__(self, pin, suppress=False):
         self._lf = False
         self._ld = False  # Delay_ms instance for long press
         self._dd = False  # Ditto for doubleclick
-        self.sense = pin.value()  # Convert from electrical to logical value
+        self.sense = pin.value() if sense is None else sense  # Convert from electrical to logical value
```

Comment:
Considered using Pin.pull() to get the pin set up, but in many cases (mine included) the internal pull up resistors are not configured, in favour of external resistors.